src/features/documentation.ts

```diff
@@ -2,29 +2,30 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-'use strict';
-
 
-const _regExp = /<(\S*?).*?>((.|\r|\n)*?)<\/\1>/;
+'use strict';
 
-/**
- * remove xml-tags from string
- */
-export function plain(doc: string): string {
+const summaryStartTag = /<summary>/i;
+const summaryEndTag = /<\/summary>/i;
```

Comment:
No. They're case *in*sensitive

![image](https://cloud.githubusercontent.com/assets/116161/14728164/503b0f02-07e7-11e6-96cc-4e2ddb3a983f.png)